src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorTests.cs

```diff
@@ -768,5 +768,41 @@ internal partial class JsonContext : JsonSerializerContext
             Compilation compilation = CompilationHelper.CreateCompilation(source);
             CompilationHelper.RunJsonSourceGenerator(compilation);
         }
+
+#if false
```

Comment:
Doing this both because skipped tests seem to not be allowed, and because the test needs to reference LanguageVersion.CSharp12, which isn't in the runtime's test harness yet.